src/PileupTrack.js

```diff
@@ -214,6 +214,7 @@ const PileupTrack = (HGC, ...args) => {
       this.drawnAtScale = HGC.libraries.d3Scale.scaleLinear();
       this.prevRows = [];
       this.coverage = {};
+      this.coverageSamplingDistance = 1;
```

Comment:
What exactly is the coverage sampling distance? A comment in the code would be helpful for future updates.

Reply:
That's the bp distance for which the samples are chosen for the coverage. E.g. if it is 5, we might compute the coverage only at positions 1.000.000, 1.000.005, 1.000.0010, ...

Reply:
Yeah, then I think we definitely need some visual indication that the data is sampled because even though the coverage looks similar to non sampled, the risk of missing variants in the coverage plot grows by quite a bit?

Actually, for that assessment I'm assuming that you're showing variants in the coverage plot?

Reply:
Yes, that's true. The risk of missing variants is large. On the other hand, they are hardly visible or not visible at all in the exact case as well, since the rectangles get so thin.

Reply:
I mean the vertical coverage rectangles

Reply:
I can just make the color of the coverage track lighter when it is sampled. Would that be sufficient?

Reply:
We could also remove the variants from the coverage plot, when it is not exact (sampled). Together with a lighter color, this might be sufficient to indicate that one has to zoom in to see more details.

Reply:
Yeah, I think that makes sense 👍

Somewhat related but I've been wondering if we can't use a multivec track to store the positions of variants so that we can do genome-wide views of variation. Haven't had too much time to work on it though. Anyway, maybe something to consider for the future.

Reply:
That sounds definitely interesting. Are there tools that are doing that?

Reply:
No, we'd have to build one.